.github/workflows/tests.yml

```diff
@@ -21,6 +21,8 @@ jobs:
         pip install wheel
         pip install -r requirements.txt
         pip install -r requirements-dev.txt
+    - name: Install Spanish Locale
+      run: sudo locale-gen es_ES.UTF-8
```

Comment:
And I suppose we don't need this anymore?

Reply:
yep

Reply:
gone